docs/.vuepress/theme/styles/index.styl

```diff
@@ -186,6 +186,23 @@ h5 {
   font-size: 1em;
 }
 
+h1 a.header-anchor::before,
```

Comment:
Wouldn't it be better to shorten this into:
```
h1, h2, h3, h4, h5 {
  a.header-anchor::before {
  content: '#';
  visibility: 'hidden';
  }

  &:hover {
    a.header-anchor::before {
      visibility: 'visible'
    }
  }
}
```

Reply:
True, I forgot it's not a vanilla CSS file 🤦